quickstart/aspnet-core/ContosoTeamStats/Controllers/HomeController.cs

```diff
@@ -1,29 +1,21 @@
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Logging;
 using ContosoTeamStats.Models;
-using StackExchange.Redis;
-using System.Net.Sockets;
-using System.Text;
-using System.Threading;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System.Diagnostics;
+using System.Threading.Tasks;
 
 namespace ContosoTeamStats.Controllers
 {
     public class HomeController : Controller
     {
         private readonly ILogger<HomeController> _logger;
-        private static IConfiguration Configuration { get; set; }
+        private readonly RedisConnection _redisConnection;
 
-        public HomeController(ILogger<HomeController> logger, IConfiguration configuration)
+        public HomeController(ILogger<HomeController> logger, RedisConnection redisConnection)
```

Comment:
>RedisConnection redisConnection [](http://example.com/codeflow?start=62&length=31)

where does this come from? Wouldn't we have to configure DI to get this injected?